tools/modules/software/module_plexmediaserver.sh

```
@@ -1,15 +1,17 @@
-module_options+=(
-	["module_plexmediaserver,author"]="@schwar3kat"
+software_options+=(
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)